osu.Game/Graphics/DrawableDate.cs

```diff
@@ -58,8 +104,18 @@ private void updateTimeWithReschedule()
 
         public override bool HandleMouseInput => true;
 
-        private void updateTime() => Text = date.Humanize();
+        // if date's format is specified
+        private void updateTime() => Text = dateFormat != "" ?
```

Comment:
`string.IsNullOrEmpty` here and below, also too many comments, pretty self explanatory what all this does.